graphql_jwt/decorators.py

```diff
@@ -62,14 +64,14 @@ def check_perms(user):
 
 def token_auth(f):
     @wraps(f)
+    @setup_jwt_cookie
     def wrapper(cls, root, info, password, **kwargs):
         def on_resolve(values):
             user, payload = values
             payload.token = get_token(user, info.context)
```

Comment:
Don't we need to get the token from cookie in get_token?

Reply:
Not here, this is to request a new token.

Reply:
I was talking about adding the logic inside get_token function. Though it seems like not the right place. I did it like that: https://github.com/mtszsobczak/django-graphql-jwt/commit/928fc4c3ccc0378a00fb077c3b0443fc6e3d099b looks like it is working ;)

Reply:
Yeah, that should work :)